backend/src/api/events.js

```diff
@@ -0,0 +1,196 @@
+const express = require('express')
+const mongodb = require('mongodb')
+const router = express.Router()
+const { errorWrap } = require('../middleware')
+const { Event } = require('../models')
+
+// get all events
+router.get(
+  '/',
+  errorWrap(async (req, res) => {
+    let events = await Event.find()
+    res.json({
+      code: 200,
+      message: '',
+      result: events,
+      success: true
+    })
+  })
+)
+
+// get one event
+router.get(
+  '/:event_id',
+  errorWrap(async (req, res) => {
+    const event = await Event.findById(req.params.event_id)
+    res.json({
+      code: 200,
+      message: '',
+      result: event,
+      success: true
+    })
+  })
+)
+
+// create a new event
+router.post(
+  '/',
+  errorWrap(async (req, res) => {
+    const data = req.body
+    let response = 'Event Created Successfully'
+    let code = 404
```

Comment:
From what I can tell, these variables (`code`, `success`, `eventName`, ...) are not changed. Use `const` instead of `let`.